src/App.js

```diff
@@ -1,5 +1,51 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+import { randomNum } from './utils/randomNum.js'
+import { userInput } from './utils/userInput.js'
+import { restartGame } from './utils/restartGame.js'
+import { resultStrikeBall } from './utils/resultStrikeBall.js'
+import { message } from './const/message.js'
 class App {
-  async play() {}
+	async play() {
+		//시작
+		MissionUtils.Console.print('숫자 야구 게임을 시작합니다.')
+
+		const randomArr = randomNum()
+
+		while (true) {
+			//사용자 입력
+			const userInputNumber = await userInput()
+
+			//컴퓨터 결과 출력
+			const { strike, ball } = resultStrikeBall(randomArr, userInputNumber)
+
+			if (strike === 3) {
+				MissionUtils.Console.print('3' + message.STRIKE)
+				MissionUtils.Console.print(message.END)
+				break
+			}
+
+			if (strike > 0 && ball === 0) {
+				MissionUtils.Console.print(strike + message.STRIKE)
+			}
+			if (ball > 0 && strike === 0) {
+				MissionUtils.Console.print(ball + message.BALL)
+			}
+			if (ball > 0 && strike > 0) {
+				MissionUtils.Console.print(
+					ball + message.BALL + ' ' + strike + message.STRIKE,
+				)
```

Comment:
```suggestion
MissionUtils.Console.print(`${ball + message.BALL} ${strike + message.STRIKE}`);
```
이렇게 템플릿 리터럴 문법을 사용하면 가독성에 좋을 것 같아요!

Reply:
간결해보이고 좋은 추천인거같습니다! 감사합니당